error when trying to compile: undefined reference to symbol 'LINES'

Compiling dcc at revision 0954ad5f17d0767f2f2129b6687e2b5d7326834a gives the following error message:

```
u@x220 ~/D/p/d/src> make -B
bclang -Wall -g -c chklib.c -o chklib.o
clang -Wall -g -c parser.c -o parser.o
clang -Wall -g -c bundle.c -o bundle.o
clang -Wall -g -c error.c -o error.o
clang -Wall -g -c backend.c -o backend.o
clang -Wall -g -c dcc.c -o dcc.o
clang -Wall -g -c perfhlib.c -o perfhlib.o
clang -Wall -g -c dataflow.c -o dataflow.o
clang -Wall -g -c idioms.c -o idioms.o
clang -Wall -g -c reducibl.c -o reducibl.o
clang -Wall -g -c comwrite.c -o comwrite.o
clang -Wall -g -c locident.c -o locident.o
clang -Wall -g -c graph.c -o graph.o
clang -Wall -g -c frontend.c -o frontend.o
clang -Wall -g -c procs.c -o procs.o
clang -Wall -g -c ast.c -o ast.o
clang -Wall -g -c fixwild.c -o fixwild.o
clang -Wall -g -c symtab.c -o symtab.o
clang -Wall -g -c hlicode.c -o hlicode.o
clang -Wall -g -c udm.c -o udm.o
clang -Wall -g -c scanner.c -o scanner.o
clang -Wall -g -c control.c -o control.o
clang -Wall -g -c proplong.c -o proplong.o
clang -Wall -g -c disassem.c -o disassem.o
disassem.c:989:29: warning: cast to 'PPROC' (aka 'struct _proc *') from smaller
      integer type 'uint32_t' (aka 'unsigned int') [-Wint-to-pointer-cast]
                    setProc((PPROC)pc[icCur].ll.immed.op);
                            ^
1 warning generated.
clang chklib.o parser.o bundle.o error.o backend.o dcc.o perfhlib.o dataflow.o idioms.o reducibl.o comwrite.o locident.o graph.o frontend.o procs.o ast.o fixwild.o symtab.o hlicode.o udm.o scanner.o control.o proplong.o disassem.o -o dcc -lncurses
/usr/bin/ld: disassem.o: undefined reference to symbol 'LINES'
```

Any idea where LINES is defined? Is it an enum?

Cheers /u

--- a/src/Makefile
+++ b/src/Makefile
@@ -1,6 +1,6 @@
 CC = clang
 CFLAGS += -Wall -g
-LDFLAGS += -lncurses
+LDFLAGS += `pkg-config --libs ncurses`
 
 SOURCES  := $(wildcard *.c)
 OBJECTS  := $(SOURCES:.c=.o)

